plugins/DevicesDetection/Commands/WarmDeviceDetectorCache.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+
+namespace Piwik\Plugins\DevicesDetection\Commands;
+
+use Piwik\DeviceDetector\DeviceDetectorCacheEntry;
+use Piwik\DeviceDetector\DeviceDetectorFactory;
+use Piwik\Plugin\ConsoleCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WarmDeviceDetectorCache extends ConsoleCommand
+{
+    const OPTION_INPUT_FILE = 'input-file';
+    const OPTION_SKIP_HEADER = 'skip-header-row';
+    const OPTION_ROWS_TO_PROCESS = 'count';
+
+    private static $userAgentsPatternsToIgnore = array(
+        '/Amazon-Route53-Health-Check-Service[.]*/'
+    );
+
+    private $numCacheEntriesWritten = 0;
+
+    protected function configure()
+    {
+        $this->setName('devicedetector:warmcache');
+        $this->setDescription(
+            'Populate the device detector cache with commonly used useragent strings, as provided in the input file.');
+        $this->addArgument(self::OPTION_INPUT_FILE, InputArgument::REQUIRED, 
+            'CSV file containing list of useragents to include');
+        $this->addOption(
+            self::OPTION_ROWS_TO_PROCESS,
+            null,
+            InputArgument::OPTIONAL,
+            'Number of rows to process',
+            0
+        );
+        $this->addOption(
+            self::OPTION_SKIP_HEADER,
+            null,
+            InputArgument::OPTIONAL,
+            'Whether to skip the first row',
+            true);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->clearCacheDirectory();
```

Comment:
Can you add an option for this maybe? Eg when core ships say 50 cached entries, then we might want to add more entries but not remove the core entries